ForceFeedback/ForceFeedback.Rules/MethodTooLongTextAdornment.cs

```diff
@@ -114,23 +102,41 @@ private void OnTextBufferChanged(object sender, TextContentChangedEventArgs e)
 
             _lastCaretBufferPosition = caretPosition + change.NewLength - 1;
 
-            if (_numberOfKeystrokes >= longMethodOccurence.LimitConfiguration.NoiseDistance)
-            {
-                if (!_view.TextBuffer.CheckEditAccess())
-                    throw new Exception("Cannot edit text buffer.");
+            if (_numberOfKeystrokes < longMethodOccurence.LimitConfiguration.NoiseDistance) return;
 
-                var textToInsert = "⌫";
+            if (!_view.TextBuffer.CheckEditAccess())
+                throw new Exception("Cannot edit text buffer.");
 
-                var edit = _view.TextBuffer.CreateEdit(EditOptions.None, null, "ForceFeedback");
-                var inserted = edit.Insert(change.NewPosition + change.NewLength, textToInsert.ToString());
+            const string textToInsert = "⌫";
 
-                if (!inserted)
-                    throw new Exception($"Cannot insert '{change.NewText}' at position {change.NewPosition} in text buffer.");
+            var edit = _view.TextBuffer.CreateEdit(EditOptions.None, null, "ForceFeedback");
+            var inserted = edit.Insert(change.NewPosition + change.NewLength, textToInsert);
 
-                edit.Apply();
+            if (!inserted)
+                throw new Exception($"Cannot insert '{change.NewText}' at position {change.NewPosition} in text buffer.");
 
-                _numberOfKeystrokes = 0;
-            }
+            edit.Apply();
+
+            _numberOfKeystrokes = 0;
+        }
+
+        private bool InteresstingChangedOccured(TextContentChangedEventArgs e)
```

Comment:
Code documentation is missing. Please describe the general intent of the methods, parameters etc.